core/indexer/client/errors.go

```diff
@@ -0,0 +1,12 @@
+package client
+
+import "errors"
+
+// ErrCouldNotCreatePolicy -
+var ErrCouldNotCreatePolicy = errors.New("could not create policy")
+
+// ErrNoElasticUrlProvided signals that was not provided a url to the elasticsearch database
```

Comment:
`// ErrNoElasticUrlProvided signals that the url to the elasticsearch database hasn't been provided`

Reply:
done.